src/main/resources/lib/credentials/select.jelly

```diff
@@ -75,6 +80,29 @@
     </j:if>
     <div field-disabled="${paramValue?'true':null}"
          class="credentials-select-content-select ${paramValue?'credentials-select-content-inactive':'credentials-select-content-active'}">
+      <j:choose>
+        <j:when test="${includeUser and attrs.includeUserCaution}">
+          <div class="include-user-credentials">
+            <label field-disabled="true">
+              <input type="checkbox" name="includeUser"/>
+              ${%includeUserCredentials}
+            </label>
+            <span class="warning user-credentials-caution" hidden="hidden">
+              ${%userCredentialsCaution}
+            </span>
+            <a href="#" class="help-btn"><l:icon class="icon-help icon-sm" alt="${%Help}"/></a>
+            <div class="help-content" hidden="hidden">
+              <p>${%includeUserCredentialsHelp}</p>
+              <p class="from-plugin">
+                (from <a href="https://wiki.jenkins.io/display/JENKINS/Credentials+Plugin">Credentials Plugin</a>)
+              </p>
+            </div>
```

Comment:
The `p` should be `div` to avoid adding too much space in the help panel. Otherwise the current style is very accurately similar to the rest of the application, good job!

<details>

<summary>Screenshot comparison</summary>

With `p`:
![Screenshot_2019-08-26_091746_001](https://user-images.githubusercontent.com/2662497/63673072-b6ce7c80-c7e3-11e9-9df7-c00dc9634cda.png)

With `div`:
![Screenshot_2019-08-26_091756_001](https://user-images.githubusercontent.com/2662497/63673081-bb933080-c7e3-11e9-9c8a-9ccb8e840004.png)

</details>

It's interesting to notice that 90% of the other help are using `div` but there are still some using `p` which look weird in comparison.

Tested with Chrome, Firefox, IE11, Edge: it works fine.